docs/source/guide/index.md

```diff
@@ -81,8 +81,30 @@ The component parts of Label Studio are available as modular extensible packages
 | [Data Manager](https://github.com/heartexlabs/dm2) | JavaScript web app using [React](https://reactjs.org/) | Manage data and tasks for labeling. |
 | [Machine Learning Backends](https://github.com/heartexlabs/label-studio-ml-backend) | Python | Predict data labels at various parts of the labeling process. |
 
+The following figure shows the different components in Label Studio architecture.
 <br>
-<div style="margin:auto; text-align:center;"><img src="/images/ls-modules-scheme.png" style="opacity: 0.8"/></div>
+<div style="margin:auto; text-align:center;"><img src="/images/ls-modules-scheme.png" style="opacity: 0.8" class="gif-border"/></div>
+<i>Figure 1: Overview of Label Studio architecture.</i>
+
+
+<i class='ent'/></i> The following figure shows the labeled UI elements in Workspace view.
```

Comment:
We should remove enterprise badge everywhere and write what elements are in enterprise separately in each image.

Reply:
I don't have the original source of the three images. I will ask [ricardo@heartex.com](mailto:ricardo@heartex.com) to make the magic happen here:)

Reply:
@makseq can you provide the ENT badges for Ricardo to provide the original source of three images?

Reply:
I've called out the ENT features as a list and hope this is good for now. We can add more enhancements to this page when the new docs site is published.